test/functional/feature_assumeutxo.py

```diff
@@ -13,17 +13,12 @@
 
 Interesting test cases could be loading an assumeutxo snapshot file with:
 
-- TODO: Valid snapshot file, but referencing a snapshot block that turns out to be
-      invalid, or has an invalid parent
 - TODO: Valid snapshot file and snapshot block, but the block is not on the
       most-work chain
 
 Interesting starting states could be loading a snapshot when the current chain tip is:
 
-- TODO: An ancestor of snapshot block
```

Comment:
I think it's a bit too verbose for the log but I added a comment. And I think it will be confusing to explicitly say everywhere in the normal case that the tip is an ancestor of the snapshot. But it's documented in one place now at least.

Reply:
Yup, seems like its the most likely use case for assumeutxo, so I could see how that could be quite verbose.